zebra-chain/src/parameters/network/testnet.rs

```diff
@@ -786,4 +817,14 @@ impl Network {
             self.post_nu6_funding_streams()
         }
     }
+
+    /// Returns true if this network should allow transactions with transparent outputs
+    /// that spend coinbase inputs
+    pub fn should_allow_unshielded_coinbase_spends(&self) -> bool {
+        if let Self::Testnet(params) = self {
+            params.should_allow_unshielded_coinbase_spends()
```

Comment:
We could (`is_regtest()` should be very quick to return false) but I wanted to make this configurable for custom Testnets later.

Reply:
Will this implementation not make it available to the default Testnet then? I don't think it's a big deal, but it would then be trivial to configure Zebra so that it doesn't follow the consensus rules on Testnet.

Reply:
> Will this implementation not make it available to the default Testnet then?

It will if someone changes the `false` to `true` in the Default impl for `ParametersBuilder`.

> but it would then be trivial to configure Zebra so that it doesn't follow the consensus rules on Testnet

Unfortunately I think this is already trivial on Testnet and Mainnet, and in zcashd as well, but zcashd has a mechanism for dropping misbehaving peer connections which should mitigate the issue. I think Zebra will need to have a similar mechanism before it becomes the only node implementation on the network.